src/layman/common/util.py

```diff
@@ -20,4 +20,5 @@ def clear_publication_info(info):
             del info[key]
         except KeyError:
             pass
+    info['updated_at'] = info['updated_at'].isoformat()
```

Comment:
GET Workspace Layers and GET Workspace Maps should also serialize updated_at attributes